monitor_state_dict.py

```diff
@@ -0,0 +1,110 @@
+"""
+Contains the definitions for monitor.state.
+
+This is essentially a stopgap while OOP state is worked on.
+"""
+from __future__ import annotations
+
+from typing import MutableMapping, TYPE_CHECKING, Any, DefaultDict, Dict, List, Literal, Optional, Set, Tuple, TypedDict
+
+
+class MonitorStateDict(TypedDict):
+    """Top level state dictionary for monitor.py."""
+
+    # Game related
+    GameLanguage:           Optional[str]               # From `Fileheader`
+    GameVersion:            Optional[str]               # From `Fileheader`
+    GameBuild:              Optional[str]               # From `Fileheader`
+    Horizons:               Optional[bool]                        # Does the player have Horizons?
```

Comment:
I'm not seeing how defaulting this to `False`, so never `None` would cause an issue.  Currently it's literally only ever set from `LoadGame`, which we need in order to be doing anything with it anyway.